source/install/self-managed-changelog.md

```diff
@@ -160,7 +160,7 @@ Mattermost v7.1.0 contains low severity level security fixes. [Upgrading](https:
 ### Highlights
 
 #### Insights (Beta) (Enterprise and Professional)
- - Added workplace insights consisting of usage and behavior data, which helps Enterprises further increase productivity of their employees through Mattermost functionality.
+ - Added workplace insights consisting of usage and behavior data, which helps Enterprises further increase productivity of their employees through Mattermost functionality. This feature can be disabled using the ``MM_FeatureFlag_InsightsEnabled`` feature flag.
```

Comment:
I am not sure this clarifies how _exactly_ can it be disabled via the feature flag. The environment variable needs to be set to false. 

@cwarnermm - do we want to document that here? Or is there somewhere specific in general where feature flags and how to disable them are documented?

Reply:
There are some details here https://developers.mattermost.com/contribute/server/feature-flags/#faq, but probably doesn't have all the details.

Reply:
Yes, that is there, I am just not sure if customers are aware of that since that's a developer oriented page. I'll be fine if we link to that page. Also, the feature flag should be `MM_FEATUREFLAGS_INSIGHTSENABLED`. Note the "s" at end of "flag".

Reply:
I don't believe there is any content available on docs.mm.com today that describes how to manage feature flags. The feature flag content on dev.mm.com that Amy linked to is likely the most comprehensive information we have published across our information assets.

The docs don't have feature flag content because feature flags are in place to roll out new functionality, and are subsequently removed once functionality is verified via a Cloud release. Instead of documenting feature flags, the product documentation includes all configuration settings that disable product functionality.

I'm 0/5 on where in the product documentation we should add details on disabling Insights, but here are a couple of  options if the changelogs don't feel like the right place:
- On the Insights page itself: https://docs.mattermost.com/welcome/insights.html
- In our configuration settings documentation for system admins: https://docs.mattermost.com/configure/configuration-settings.html

I'm new to disabling functionality via feature flags. What does this process look like in terms of actionable steps?
- Does a self-hosted system admin update the ``config.json`` file with an environment variable?
- What would Cloud system admins need to do to disable Insights?
- Do admins need to perform extra steps for Mattermost clients including desktop and mobile (v1/v2)?
- Is there a System Console UI admins can use to disable feature flags? Or is that read-only?
- Once Insights is disabled, can admins re-enable at a later date?

Reply:
Understood. @amyblais - let's just avoid the word feature flag and say "This feature can be disabled by setting the `MM_FEATUREFLAGS_INSIGHTSENABLED` environment variable to `false`." It clearly says what needs to be done, and avoids any further rabbit holes.

Reply:
I've applied the suggested edit 👍